File: 프로그래머스/lv2/42584. 주식가격/주식가격.py
def solution(prices):
    stack = []
    for i in range(len(prices)-1):
        second = 0
        for j in range(i, len(prices)-1):
            if (prices[i] <= prices[j]): second += 1
            else: break
        stack.append(second)
    stack.append(0)
    return stack

File: 프로그래머스/2/42584. 주식가격/주식가격.js
const solution = prices => {
    const answer = []
    for (let i=0; i<prices.length - 1; i++) {
        let cnt = 0
        for (let j=i; j<prices.length - 1; j++) {
            
            if (prices[i] <= prices[j]) cnt++
            else break
        }
        answer.push(cnt)
    }
    return [...answer, 0]
}